lib/srv/discovery/fetchers/db/aws_redshift.go

```diff
@@ -129,3 +129,16 @@ func getRedshiftClusters(ctx context.Context, redshiftClient redshiftiface.Redsh
 	)
 	return clusters, trace.Wrap(libcloudaws.ConvertRequestFailureError(err))
 }
+
+// MatchingLabels returns the labels that the fetcher is matching.
+func (f *redshiftFetcher) MatchingLabels() types.Labels {
+	return f.cfg.Labels
+}
+
+func (f *redshiftFetcher) MatchesResource(r types.ResourceWithLabels) bool {
+	db, ok := r.(types.Database)
+	if !ok {
+		return false
+	}
+	return db.IsRedshift() && db.GetAWS().Redshift.ClusterID != "" && !db.GetAWS().IsEmpty()
```

Comment:
nit:
```suggestion
	return db.IsRedshift()
```
Sorry for the confusing db types. RedshiftServerless is not Redshift. They are two different types.